modules/index.js

```diff
@@ -94,47 +104,40 @@ bookForm.addEventListener('submit', (e) => {
 
 window.addEventListener('DOMContentLoaded', book.showBook());
 
-const navlist = document.querySelector('#show-list-button');
-const navadd = document.querySelector('#add-new-button');
-const navcon = document.querySelector('#contact-button');
-
-const list = document.querySelector('#list');
-const addNew = document.querySelector('#add');
-const connav = document.querySelector('#contact');
-
-function listfun(e) {
+const listfun = (e) => {
   if (e.currentTarget === navlist) {
     list.style.display = 'flex';
     addNew.style.display = 'none';
     connav.style.display = 'none';
   }
-}
+};
 
-function addfun(e) {
+const addfun = (e) => {
   if (e.currentTarget === navadd) {
     list.style.display = 'none';
     addNew.style.display = 'flex';
     connav.style.display = 'none';
   }
-}
+};
 
-function confun(e) {
+const confun = (e) => {
   if (e.currentTarget === navcon) {
     list.style.display = 'none';
     addNew.style.display = 'none';
     connav.style.display = 'flex';
   }
-}
+};
 
 navlist.addEventListener('click', listfun);
 
 navadd.addEventListener('click', addfun);
 
 navcon.addEventListener('click', confun);
 
-// eslint-disable-next-line no-unused-vars
 const displayTime = () => {
-  document.getElementById('current-date').innerText = new Date().toLocaleString();
+  document.getElementById('current-date').innerText = DateTime.now().toFormat(
+    "HH 'hours and' mm 'minutes'",
+  );
```

Comment:
- [ ] According to the project requirements, the luxon library should be used to display the current `date and time` and what is displayed is only the time in 24hr format, please update it to show the current date and time.

Reply:
Everything is alright now. you can check the update on the readme now. and in the index.js file